tools/hookgen/makefile

```diff
@@ -30,4 +30,11 @@ MODULE_INCLUDES += $(OMR_PUGIXML_DIR)
 vpath %.cpp $(OMR_PUGIXML_DIR)
 vpath %.c $(OMR_PUGIXML_DIR)
 
+# HookGen build failed with pase vac13 and it only run on AIX.
+# So, we use the default vac on AIX build system
+ifdef I5_VERSION
```

Comment:
I am not sure that it is best to change the compiler here.  Can you provide more details what is wrong with the default compiler?